docs/conf.py

```diff
@@ -66,6 +66,8 @@
     "scikit-sparse": ("https://scikit-sparse.readthedocs.io/en/latest/", None),
     "scipy": ("https://docs.scipy.org/doc/scipy/reference/", None),
     "pot": ("https://pythonot.github.io/", None),
+    "jaxopt": ("https://jaxopt.github.io/", None),
```

Comment:
The `BFGS` reference from `jaxopt` doesn't render, can you please try https://jaxopt.github.io/stable/?

Reply:
That fixed it:

![image](https://user-images.githubusercontent.com/707462/217890463-75858094-18e0-45a6-b294-6664c9b6604e.png)